code-challenges.test.js

```diff
@@ -61,34 +130,111 @@ var nums1 = [3, 56, 90, -8, 0, 23, 6]
 var nums2 = [109, 5, 9, -59, 8, 24]
 // Expected output: [-59, 109]
 
+describe("When minAndMax is called on an array,", () => {
+
+    it("returns an array of the minimum and maximum numbers in that order.", () => {
+
+        const nums1 = [3, 56, 90, -8, 0, 23, 6];
+
+        actResult = minAndMax(nums1);
 
+        expect(actResult).toEqual([-8, 90]);
+    })
 
+    it("returns an array of the minimum and maximum numbers in that order.", () => {
 
+        const nums2 = [109, 5, 9, -59, 8, 24];
+
+        actResult = minAndMax(nums2);
+
+        expect(actResult).toEqual([-59, 109]);
+
+    })
+})
 
 // b) Create the function that makes the test pass.
 
+const minAndMax = (array) => {
 
+    let newArr = [];
 
+    newArr.push(Math.min.apply(Math, array));
```

Comment:
there might be another way of doing this without needing to use `.apply`. Still works though!